Release new DAMs (#95)
CloudCannon/platform-documentation: articles/creating-a-cloudflare-r2-dam.mdx

---
_uuid: "7b8ac25c-420f-4d3b-9a16-2ffc1c73807b"
_created_at: "2023-05-23 03:31:41 +0000"
title: "Creating a Cloudflare R2 DAM"
nav_title: "Cloudflare R2 DAM"
published: true
image: "/documentation/static/CloudCannonDocumentationog.jpg"
description: Set up a DAM with Cloudflare R2 and link it to your CloudCannon Organization
tags: []
related_articles: ["integrating-your-dam-with-cloudcannon"]
related_links: [{"name":"Cloudflare R2","url":"https://www.cloudflare.com/products/r2/"}]
---

> Cloudflare R2 is an inexpensive cloud storage service built on top of Amazon's S3 service. 

Reviewer: ```suggestion
Cloudflare R2 is an inexpensive cloud storage service built to be compatible with Amazon's S3 interface. 
```
